src/java.desktop/share/classes/java/awt/color/CMMException.java

```diff
@@ -42,6 +42,7 @@
 
     /**
      * Use serialVersionUID from JDK 1.2 for interoperability.
+     * Make some change to make jcheck happy.
      */
     private static final long serialVersionUID = 5775558044142994965L;
```

Comment:
``private static long serialVersionUID = 5775558044142994965L;

necessarily be final?